package.json

```diff
@@ -17,7 +17,6 @@
     "lint-staged": {
         "*.ts": "yarn lint-fix"
     },
-    "packageManager": "yarn@berry",
```

Comment:
> Is it strictly required to use `yarn@berry`? I used my local `yarn` version but had to remove the following line to be able to commit?
> 
> ```
> "packageManager": "yarn@berry",
> ```


Lets avoid changing the configuration :upside_down_face: 
What is the error when you try to commit without removing this line? :thinking: 
(What is the result of `yarn -v` when you run it in the repo? :thinking:)

Reply:
```
yarn -v
3.2.1
```

Usage Error: Invalid package manager specification in package.json; expected a semver version

$ yarn ...

Reply:
Hmm, I can't reproduce it.
But it seems it is not required, and mostly a feature for corepack. Maybe if we can change the line to "yarn@3.2.1" (as that seems to be the embedded version) that would be nice. :upside_down_face: 
Otherwise if that doesn't work for you, we can remove the line. :shrug:

Reply:
```
"packageManager": "yarn@3.2.1",
```

This works find for me. Added it.